### DISCUSSION TITLE
The minimum support is node: 16, so should I not use the unsupported specs in 16.0.0?

### DISCUSSION THREAD

**Discussion by yujeong-jeon** [NaverPayDev/hidash#127] [Q&A] [answered]
I would like to ask how far ES function can be used when implementing it.

@yceffort-naver 님 구현할 떄 어디까지 ES기능을 활용할 수 있을지 문의드립니다.

**Comment by yceffort-naver** [comment, marked as answer]
- 16 기준으로 하고 있습니다. (생각난김에 browserlist 에 node16 넣어야겠군요)
- 현재는 해당 browserlist 기준으로 폴리필 하나도 안넣고 구현하는걸 목적으로 하고 있습니다. 

https://github.com/NaverPayDev/hidash/blob/1b475a56c7c51722eb018b3bc00f1602baabf62a/vite.config.mts#L20-L26

cc) @keemhyunseok @byhhh2 

- 이렇게 설정해두었는데, 혹시 core-js가 폴리필 넣어야 한다고 하면 에러가 날거예요... 에러 날때 위 코드 처럼 무시할지, 혹은 꼮 넣어야할지는 그시점에 한번 검토해봐야 할것 같습니다.

**Reply by yceffort-naver** [reply]
https://github.com/NaverPayDev/hidash/issues/129

**Reply by yujeong-jeon** [reply]
### Translations

It's based on Node.js 16. (I should put Node16 in the browser list now that I remember it.)
Currently, the purpose is to implement it without putting any polyphiles based on the browser list.

I've set it up like this, but if core-js says I need to put polyfill, there will be an error... I think I'll have to review it at that point whether I should ignore it like the code above or put it in when I get an error.

https://github.com/NaverPayDev/hidash/blob/1b475a56c7c51722eb018b3bc00f1602baabf62a/vite.config.mts#L20-L26
